src/main/java/com/solvd/meta/carina/gui/pages/InventoryPage.java

```diff
@@ -1,21 +1,26 @@
-package com.qaprosoft.carina.demo.gui.pages;
+package com.solvd.meta.carina.gui.pages;
 
 import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.solvd.meta.carina.gui.components.Header;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.FindBy;
 
 import java.util.List;
 
-public class InventoryPage {
+public class InventoryPage extends AbstractPage {
     @FindBy(xpath = "//div[@class='inventory_item']")
     private List<ExtendedWebElement> products;
```

Comment:
Create please Product as a component because it is repeated on many pages. It will help to reuse code